src/hydro_system.hpp

```diff
@@ -138,7 +138,8 @@ class HydroSystem : public HyperbolicSystem<problem_t> {
   // C++ does not allow constexpr to be uninitialized, even in a templated
   // class!
   static constexpr double gamma_ = HydroSystem_Traits<problem_t>::gamma;
-  static constexpr double cs_iso_ = HydroSystem_Traits<problem_t>::cs_isothermal;
+  static constexpr double cs_iso_ =
```

Comment:
since there's no substantive changes to this file, can you revert the style changes for this file?

Reply:
Done.